SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/WorkerRegion.cpp

```diff
@@ -10,30 +10,30 @@ namespace
 {
 	const float DEFAULT_WORKER_REGION_HEIGHT = 30.0f;
```

Comment:
we could also drop this HEIGHT param and just use vertical scale from a height of 0